doc/api/next_api_changes/deprecations/23449-SS.rst

```diff
@@ -0,0 +1,5 @@
+``plot3d.view_transformation``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+... is deprecated. The private functions `plot3d._view_transformation_uvw` and
```

Comment:
It is probably better to state that the code should be vendored than to point at private API.